enterprise-suite/templates/node-exporter.yaml

```diff
@@ -20,7 +18,7 @@ spec:
         app: prometheus
         component: node-exporter
     spec:
-      serviceAccountName: "default"
+      serviceAccountName: "prometheus-kube-state-metrics"
```

Comment:
I really don't fully understand what's going on here, but should this be helm template variable?  (I mean, is it okay that this serviceAccountName be enshrined for everyone in the `.yaml`?

Reply:
Don't we need to create this serviceAccount in the first place @srikanthops ?

Reply:
@jsravn  
We already have this serviceAccount, it is attached to "kube-state-metrics" 
I was piggybacking on the same serviceAccount.

We can surely create a new serviceAccount, let me do that to be on the safe side 

@marcoderama 
As mentioned above I am going to create a separate ServiceAccount and attach it to node-exporter-Deployment. 
If an empty default securityAssociation works fine with minikube, we don't need a helm template variable